nailgun/entities.py

```diff
@@ -3971,6 +4195,43 @@ def update_payload(self, fields=None):
         }
 
 
+class SmartClassParameters(
+        Entity,
+        EntityReadMixin,
+        EntitySearchMixin,
+        EntityUpdateMixin):
+    """A representation of a Smart Class Parameters."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'puppetclass': entity_fields.OneToOneField(PuppetClass),
+            'override': entity_fields.BooleanField(),
+            'description': entity_fields.StringField(),
+            'default_value': entity_fields.StringField(),
+            'hidden_value': entity_fields.BooleanField(),
```

Comment:
It contains, well, hidden value :)
Or to be more precise, it contains what is displayed instead of actual value.
```python
>>> sc_param.read().hidden_value
u'*****'
```